cypress/integration/account_settings/display/channel_display_mode_spec.js

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Account Settings > Display > Channel Display Mode', () => {
+    before(() => {
+        // 1. Go to Account Settings with "user-1"
+        cy.toAccountSettingsModal('user-1');
+    });
+
+    beforeEach(() => {
+        cy.viewport(1500, 660);
+    });
+
+    it('should render in min setting view', () => {
+        // * Check that the Display tab is loaded
+        cy.get('#displayButton').should('be.visible');
+
+        // 2. Click the Display tab
+        cy.get('#displayButton').click();
+
+        // * Check that it changed into the Display section
+        // * Check the min setting view if each element is present and contains expected text values
+        cy.get('#displaySettingsTitle').should('be.visible').should('contain', 'Display Settings');
+        cy.get('#channel_display_modeTitle').should('be.visible').should('contain', 'Channel Display');
+        cy.get('#channel_display_modeDesc').should('be.visible').should('contain', 'Fixed width');
```

Comment:
Is the default value `Full width` here? My login says `Full width` here, but I can't remember if I changed it during testing over the last couple days.

Are we looking to test defaults?

Reply:
I see the ticket asked for fixed width so my question may not be relevant, but I'm still curious. :)

Reply:
Yes the default is full width, but the user-1 in test-data has fixed width in first load

Reply:
Great, thanks for the clarification!